org.eclipse.scout.rt.mom.api/src/main/java/org/eclipse/scout/rt/mom/api/AbstractMomTransport.java

```diff
@@ -34,7 +36,25 @@ public abstract class AbstractMomTransport implements IMomTransport {
 
   private static final Logger LOG = LoggerFactory.getLogger(AbstractMomTransport.class);
 
-  protected final FinalValue<IMomImplementor> m_delegate = new FinalValue<>();
+  protected final String m_momUid = UUID.randomUUID().toString();
```

Comment:
What's the purpose of two different identifiers?
- org.eclipse.scout.rt.mom.api.AbstractMomTransport#m_momUid
- org.eclipse.scout.rt.mom.jms.JmsMomImplementor#m_momUid

Could/should the transport use getDelegate().getId() instead of an own identifier?

Reply:
Intended. ID of transport should not change during a run. ID of implementor changes every time #destroy is called on transport.